docs/content/preview/explore/query-1-performance/pg-stats.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Get column statistics using pg_stats
+linkTitle: Column statistics
+headerTitle: Column statistics using pg_stats
+headcontent: Improve queries using column level statistics of tables from pg_stats view
+menu:
+  preview:
+    identifier: pg-stats
+    parent: query-tuning
+    weight: 400
+type: docs
+---
+
+[pg_stats](../../../architecture/system-catalog#data-statistics) is a system view that provides statistical information about columns in tables. Use it to view data distributions of column values, which can help in query optimization and tuning. To generate the statistics, run the [ANALYZE](../../../api/ysql/the-sql-language/statements/cmd_analyze/) command.
+
+## Key statistics
+
+pg_stats contains information such as the following:
+
+- **null_frac**: Fraction of NULL values in a column.
+- **avg_width**: Average width (in bytes) of a column's entries.
+- **n_distinct**: Estimated number of distinct values in a column. Positive values are direct counts, while negative values indicate that the distinct count is a fraction of the total row count.
+- **most_common_vals**: Most frequently occurring values in a column.
+- **most_common_freqs**: Frequencies of the most common values.
+- **most_common_elems**: Most frequently occurring elements in an array column.
+- **most_common_elem_freqs**: Frequencies of the most elements in an array column.
+
+Run  the following examples to understand how you can use these statistics to improve queries and the data model.
+
+## Setup
+
+The examples run on any YugabyteDB universe.
+
+<!-- begin: nav tabs -->
+{{<nav/tabs list="local,anywhere,cloud" active="local"/>}}
+
+{{<nav/panels>}}
+{{<nav/panel name="local" active="true">}}
+<!-- local cluster setup instructions -->
+{{<setup/local numnodes="1" rf="1" >}}
+
+{{</nav/panel>}}
+
+{{<nav/panel name="anywhere">}} {{<setup/anywhere>}} {{</nav/panel>}}
+{{<nav/panel name="cloud">}}{{<setup/cloud>}}{{</nav/panel>}}
+{{</nav/panels>}}
+<!-- end: nav tabs -->
+
+Create a users table:
+
+```sql
+CREATE TABLE users (
+    id int,
+    name VARCHAR,
+    employed BOOLEAN,
+    PRIMARY KEY(id)
+);
+```
+
+Add some data to your table so that you can correlate it during analysis.
+
+```sql
+-- set seed for random to be repeatable
+SELECT setseed(0.5); -- to help generate the same random values
+
+-- Insert 10,000 rows into the users table
+INSERT INTO users (id, name, employed)
+SELECT
+    id,
+    CASE
+        -- 40% NULL names
+        WHEN random() < 0.4 THEN NULL
+        -- 60% non-null names with skewed distribution
+        ELSE (
+            CASE
+                -- 30% concentration of empty string
+                WHEN random() < 0.5 THEN ''
```

Comment:
```suggestion
                WHEN random() < 0.3 THEN ''
```

Reply:
suggestion is because of the comment - -- 30% concentration of empty string

Reply:
The comment is correct. 40% NULLs & 60% non-NULLs. of the 60% 50% is empty string which is 30% of the total.